docker-compose.e2e.yml

```diff
@@ -27,5 +27,6 @@ services:
       - 5555:5432
     networks:
       - shoegaze-e2e
+# I don't think you need to set a network. The default one should be fine.
```

Comment:
I didn't want two postgres servers in the same network, and the db for the app when you run locally uses the default network